platform/jobs/RetirementJob.groovy

```diff
@@ -0,0 +1,33 @@
+package testeng
+
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.JENKINS_PUBLIC_TEAM_SECURITY
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.JENKINS_PUBLIC_LOG_ROTATOR
+
+/* Retirement.groovy
+ *
+ */
+
+
+/* This is the job DSL responsible for creating the pipeline job.
+ */
+pipelineJob('retirement') {
+    description(
+        'Retire a group of users which has been queued up.'
+    )
+
+    /* Give all job control a read permissions to all edx employees. */
+    /* TODO: make this configurable */
+    authorization JENKINS_PUBLIC_TEAM_SECURITY.call(['edx'])
```

Comment:
@pwnage101 in the past, the thinking has been that these jobs are first and foremost job configurations specific for edx infrastructure, but that they can be modified to suit others needs. That being said, perhaps we should make these easier to parameterize. It would be nice to pass values (in this case, the team name) in to the seed job (or make it available as a variable or something) so that members of the open source community don't have to fork or manually edit the DSL each time they wish to seed.

Reply:
As for your other question about the auth strategy, it would probably be something like this (using dummy names):

// Make sure that you block the inherited security scheme (it may be more permissive than what you want)
blocksInheritance(true)

// Either manually feed the users you want to grant control to OR create github group(s) and put them in this list
List membersWithFullControl = ['employee1', 'employee2']                                                                                                                                                                                                                                                                                                    
membersWithFullControl.each { emp ->
    permissionAll(emp)
}
// individually give 'discover' (find the job on the Jenkins job list/hit the URL) and 'read' (view logs/artifacts of the build) to customer support
permission('hudson.model.Item.Read', 'customer-support')
permission('hudson.model.Item.Discover', 'customer-support')